locale/hu.json

```diff
@@ -31,256 +31,256 @@
             },
             "actions": {
-                "title": "Player Actions",
-                "command_sent": "Command sent!",
+                "title": "Játékos műveletek",
+                "command_sent": "Parancs elküldve!",
                 "moderation": {
-                    "title": "Moderation",
+                    "title": "Moderálás",
                     "options": {
-                        "dm": "DM",
-                        "warn": "Warn",
-                        "kick": "Kick",
-                        "set_admin": "Give Admin"
+                        "dm": "Privát üzenet",
+                        "warn": "Figyelmeztetés",
+                        "kick": "Kirúgás",
+                        "set_admin": "Admin adás"
                     },
                     "dm_dialog": {
-                        "title": "Direct Message",
-                        "description": "What is the reason for direct messaging this player?",
-                        "placeholder": "Reason...",
-                        "success": "Your DM has been sent!"
+                        "title": "Közvetlen üzenet",
+                        "description": "Mivel kapcsolatban üzensz a játékosnak?",
+                        "placeholder": "Indok...",
+                        "success": "Üzenet elküldve!"
                     },
                     "warn_dialog": {
-                        "title": "Warn",
-                        "description": "What is the reason for direct warning this player?",
-                        "placeholder": "Reason...",
-                        "success": "The player was warned!"
+                        "title": "Figyelmeztetés",
+                        "description": "Milyen indokkal figyelmezeteted a játékost?",
+                        "placeholder": "Indok...",
+                        "success": "A játékos figyelmeztetve!"
                     },
                     "kick_dialog": {
-                        "title": "Kick",
-                        "description": "What is the reason for kicking this player?",
-                        "placeholder": "Reason...",
-                        "success": "The player was kicked!"
+                        "title": "Kirúgás",
+                        "description": "Milyen okból rúgod ki a játékost?",
+                        "placeholder": "Indok...",
+                        "success": "Játékos kirúgva!"
                     }
                 },
                 "interaction": {
-                    "title": "Interaction",
+                    "title": "Interakció",
                     "options": {
-                        "heal": "Heal",
-                        "go_to": "Go to",
-                        "bring": "Bring",
-                        "spectate": "Spectate",
-                        "toggle_freeze": "Toggle Freeze"
+                        "heal": "Gyógyítás",
+                        "go_to": "TP játékoshoz",
+                        "bring": "Megidézés",
```

Comment:
My suggestion would be `Játékos idehozása/ideteleportálása`.

Reply:
but seeing from the screenshots you attached, this could overflow the buttons so it might be better to just leave it as is.

Reply:
Unfortunately it overflow, which is why I had to compromise and use the shortest possible solution.